components/calendar_components/TaskList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Task from "./Task";
+
+export default function TaskList({ tasks, toggleTask }) {
+  return (
+    <div className="task-list">
+      <div className="lefttasks">
+        {tasks.filter((task) => !task.complete).length} Aufgaben übrig
+      </div>
+      <div className="title_tasklist">
+        <h2>To-Do-List</h2>
+      </div>
+      <div className="back">
+        <div className="value_task">
+          {tasks.map((task) => {
+            return <Task key={task.id} toggleTask={toggleTask} task={task} />;
+          })}
+        </div>
```

Comment:
Technically this could/should be a semantic `ul` list with `li` children. Just something you might want to think about 🙂